integration-tests/main_test.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"flag"
+	"fmt"
+	_ "github.com/jackc/pgx/v4/stdlib" // Importing pgx driver
+	"github.com/stretchr/testify/assert"
+	"io"
+	"log"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+)
+
+var dbIdentifier string
+var port string
+var dbConfigStr = flag.String("dbconfig", "", "JSON string of database configuration map")
+var containerConfigStr = flag.String("containerConfig", "", "JSON string of container configuration")
+
+func TestAPISuite(t *testing.T) {
+	var config ContainerConfig
+	if err := json.Unmarshal([]byte(*containerConfigStr), &config); err != nil {
```

Comment:
Can we have a default ContainerConfig. It seems that these normally don't need to be changed.

Reply:
ok ill hard code this value and remove the flag for passing it in
` -containerConfig='{
  "project_dir": "../",
  "dockerfile": "../Dockerfile",
  "image_name": "autodba:latest",
  "container_name": "autodba_test",
  "prometheus_port": "9090",
  "bff_port": "4000",
  "default_metric_period": "5",
  "warm_up_time": "60"
}'

`

Reply:
Sounds good.